packages/manager/apps/dedicated/client/app/billing/payment/method/translations/Messages_fr_CA.json

```diff
@@ -16,5 +16,6 @@
   "billing_payment_method_table_action_download_procedure": "Télécharger la procédure à retourner par voie postale",
   "billing_payment_method_table_action_description_edit": "Modifier la description",
   "billing_payment_method_table_action_set_default": "Définir ce moyen de paiement par défaut",
-  "billing_payment_method_table_action_delete": "Supprimer ce moyen de paiement"
+  "billing_payment_method_table_action_delete": "Supprimer ce moyen de paiement",
+  "billing_payment_method_credit_card_disclaimer": "L'enregistrement d'un moyen de paiement est obligatoire si vous utilisez le renouvellement automatique. Vous pouvez le modifier à tout moment. Si vous ne disposez pas d'un renouvellement automatique et que vous souhaitez supprimer votre moyen de paiement, veuillez contacter notre équipe d'assistance à la clientèle en créant un ticket."
```

Comment:
A link to ticket creation would have been nice

Reply:
it would be